tests/lua-tests/project/proj.android/app/build.gradle

```diff
@@ -1,4 +1,4 @@
-import org.apache.tools.ant.taskdefs.condition.Os
+import org.gradle.internal.os.OperatingSystem;
```

Comment:
API changed, refer to: https://stackoverflow.com/questions/11235614/how-to-detect-the-current-os-from-gradle